.yarnrc

```diff
@@ -1,2 +1,6 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
This is a weird change. Was this intentional?

Reply:
Yeah, that needs to be removed from the PR.